vsts.yml

```diff
@@ -26,71 +21,81 @@ phases:
       # better solution for checking out the commit to be built.
       (cd src/electron; git fetch origin +"${BUILD_SOURCEBRANCH}"; git checkout "${BUILD_SOURCEVERSION}")
       gclient sync --with_branch_heads --with_tags
-    name: gclient_sync
+      cd src
+      export CHROMIUM_BUILDTOOLS_PATH=`pwd`/buildtools
+      echo "##vso[task.setvariable variable=CHROMIUM_BUILDTOOLS_PATH]`pwd`/buildtools"
+    displayName: 'gclient sync'
 
   - bash: |
       cd src
-      export CHROMIUM_BUILDTOOLS_PATH=`pwd`/buildtools
-      export SCCACHE_BINARY="`pwd`/electron/external_binaries/sccache"
-      # SCCACHE_AZURE_BLOB_CONTAINER and SCCACHE_AZURE_CONNECTION_STRING are expected to be set.
-      "$SCCACHE_BINARY" --start-server
-      echo "##vso[task.setvariable variable=SCCACHE_BINARY]$SCCACHE_BINARY"
-      echo "##vso[task.setvariable variable=CHROMIUM_BUILDTOOLS_PATH]$CHROMIUM_BUILDTOOLS_PATH"
+      export SCCACHE_HELPER="`pwd`/electron/script/sccache-helper.py"
+      "$SCCACHE_HELPER" --start-server --aws-access-key "$(SCCACHE_AWS_ACCESS_KEY)" --aws-secret "$(SCCACHE_AWS_SECRET)"
```

Comment:
`$()` is for calling things, do you mean `${}`?

Also, this seems unnecessary, because sccache already reads the `SCCACHE_AWS_ACCESS_KEY` environment variable...

Reply:
`$()` is a VSTS convention to pull in a VSTS variable and is needed because VSTS won't make private variables available as an environment variable.  So either we would need to not make the S3 creds private or we need to do it this way.

Reply:
Perhaps we could use a "mapped" env var like the example here? https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=vsts&tabs=yaml%2Cbatch#secret-variables that seems better than having two 40-line python scripts, if it works

Reply:
Hmm... the "mapped" env var looks like it could work.  I'm guessing that is a new option, because I don't remember seeing that in the docs before.